pkgs/tools/networking/dd-agent/datadog-agent.nix

```
@@ -120,6 +120,7 @@ in buildGoModule rec {
     license     = licenses.bsd3;
     maintainers = with maintainers; [ thoughtpolice domenkozar ];
     # never built on aarch64-darwin since first introduction in nixpkgs
-    broken = stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64;
+    # broken = stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64;
+    broken = true;
```

How about adding a bit more context for folks unfamiliar with the details?
```suggestion
    # broken = stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64;
    # Upstream does not support Go > 1.21; for update refer to https://github.com/NixOS/nixpkgs/issues/351119
    broken = true;
```